src/create-model.jl

```diff
@@ -613,6 +614,9 @@ function create_model(
         starting_year =
             Dict((y, a) => y - graph[a].technical_lifetime[y] + 1 for y in Y for a in Ai[y])
 
+        starting_year_compact =
+            Dict((y, a) => y - graph[a].technical_lifetime[y] + 1 for y in Y for a in A) # needs to be modified, because we want a that is either investable or existing
+
```

Comment:
After looking at the changes twice, the starting year depends on the method, but it is the same parameter. So, instead of calculating the value in two different parameters, we need only one parameter, `starting_year`, whose value depends on the method.